server/plugin/utils.go

```diff
@@ -43,3 +45,24 @@ func (p *Plugin) validateTime(time string) string {
 
 	return ""
 }
+
+func IsValidUserKey(key string) (string, bool) {
+	res := strings.Split(key, "_")
+	if len(res) == 2 && res[0]+"_" == UserKeyPrefix {
+		return res[1], true
+	}
+	return "", false
+}
```

Comment:
There's no need for this function. You can use `strings.HasPrefix`.

Reply:
Discussed